src/reference/asciidoc/metrics.adoc

```diff
@@ -118,3 +120,11 @@ and
 * `tag`: `type:channel`
 * `tag`: `name:<componentName>`
 * `description`: `The remaining capacity of the queue channel`
+
+===== Disabling Meters
+
+With the <<legacy-metrics>> (which have now been removed), you could specify which integration components would collect metrics.
+By default, all meters are registered when first used.
+Now, with Micrometer, you can add `MeterFilter` s to the `MeterRegistry` to prevent some or all from being registered.
+You can filter out (deny) meters by any of the properties provided, `name`, `tag`, etc.
+See https://micrometer.io/docs/concepts#_meter_filters[Meter Filters] in the Micrometer documentation for more information.
```

Comment:
Any chances to have some sample how to disable metrics for example for `myChannel` bean?